docs/intro.rst

```diff
@@ -201,25 +227,21 @@ Installing to OS-X
       $ pip install pywbem
 
 
-.. _`Installing to Windows`:
-
-Installing to Windows
-^^^^^^^^^^^^^^^^^^^^^
+.. _`Installing to native Windows`:
 
-The pywbem tests that run on the Appveyor CI use CygWin, but for just running
-pywbem you should be able to use plain Windows or any Unix-like environment on
-Windows (such as CygWin, MinGW, Babun, or Gow).
+Installing to native Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Note that Unix-like environments on Windows bring their own Python, so double
-check that the active Python environment is the one you want to install to.
+This section describes the installation of pywbem into a native Windows
+environment (i.e. without using a UNIX-like environment such as CygWin, etc.):
```

Comment:
Add ref to section for installing under cygwin

Reply:
DONE